src/Presentation/Nop.Web/Controllers/CatalogController.cs

```diff
@@ -649,12 +649,12 @@ public ActionResult CategoryNavigation(int currentCategoryId, int currentProduct
                 _workContext.WorkingLanguage.Id,

```

Comment:
If you are not caching the result at all then you should remove "var cacheKey" because it has no use anymore